### DISCUSSION TITLE
Accessing a (guest) panel from the root of the URL

### DISCUSSION THREAD

**Discussion by wq9578** [filamentphp/filament#11417] [Help]
### Package

Panel builder

### Package Version

v3.4.4

### How can we help you?

I have installed a panel accessible freely (for guests also), following the [instructions](https://filamentphp.com/docs/3.x/panels/users#setting-up-guest-access-to-a-panel).

Is there any way to remove the guest panel name from the URL?

For example, in order to access a book title (data model `Book`, resource `BookResource`) the URL
```
https://myhost.com/books/123
```

should be sufficient instead of:

```
https://myhost.com/myguestpanel/books/123
```

I also have panels that require authentication, but one panel with global information should be accessible without specifying the panel name in the URL.

**Comment by leandrocfe** [comment]
Maybe removing the path in the panel provider?
`->path('')`

**Comment by wq9578** [comment]
Thanks, but doesn't work (tested locally):

http://127.0.0.1:8000/books/123 -> Route [filament.myguestpanel.pages.dashboard] not defined.

http://127.0.0.1:8000/books -> pause of ca. 15 seconds, then: Route [filament.myguestpanel.pages.dashboard] not defined.

http://127.0.0.1:8000/ -> redirects to http://127.0.0.1:8000/admin/login
